### PR TITLE
Modified EUI-64 format interface identifier utilities in IPv6Address LOXI-74

### PR DESCRIPTION
**Pull request by ronaldchl**
Reviewer: @andi-bigswitch @gregor-bsn

### CODE REVIEW

**Comment by bsn-abat** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com//job/loxigen_pull_req/202/
Test PASSed.

File: java_gen/pre-written/src/main/java/org/projectfloodlight/openflow/types/IPv6Address.java [hunk 1]
```diff
@@ -139,6 +142,45 @@ public boolean isMulticast() {
...
+     * <p>Refer to the followings for the conversion details:
+     * <ul>
+     * <li>RFC 7042 - Section 2.2.1
+     * <li>RFC 5342 - Section 2.2.1 (Obsoleted by RFC 7042)
+     * <li>RFC 4291 - Appendix A
+     * </ul>
+     */
+    private static long toModifiedEui64(@Nonnull MacAddress macAddress) {
+        checkNotNull(macAddress, "macAddress must not be null");
+        return    0xFFFF_FF00_0000_0000L & macAddress.getLong() << 16
```
Comment (andi-bigswitch) [inline java_gen/pre-written/src/main/java/org/projectfloodlight/openflow/types/IPv6Address.java]: would be good to add parentheses here for clarity

**Comment by andi-bigswitch** [comment]
Otherwise LGTM

File: java_gen/pre-written/src/main/java/org/projectfloodlight/openflow/types/IPv6Address.java (hunk 1 above)
Comment (ronaldchl) [inline java_gen/pre-written/src/main/java/org/projectfloodlight/openflow/types/IPv6Address.java]: Does this look OK? 381983045ff2db675e15397780cc8dfa226f2b99

Comment (ronaldchl) [inline java_gen/pre-written/src/main/java/org/projectfloodlight/openflow/types/IPv6Address.java]: And more... 6cef30d77d31b31e36b3dd31bd3e0edd0ca0b8c2

**Comment by bsn-abat** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com//job/loxigen_pull_req/203/
Test PASSed.

**Comment by bsn-abat** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com//job/loxigen_pull_req/204/
Test PASSed.

**Comment by ronaldchl** [comment]
SELF-ACK

**Comment by bsn-abat** [comment]
ABAT: START_MERGE:  Log file at http://vss1/abat/2015.11.12.0252-m.loxigen.master/abat.log

**Comment by bsn-abat** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com//job/loxigen_pull_req/205/
Test PASSed.

**Comment by bsn-abat** [comment]
ABAT: ACCEPT:  Successfully merged

In case you want to see the build log, check out :
Log file at http://vss1/abat/2015.11.12.0252-m.loxigen.master/abat.log
/cc

**Pull request merged**